modules/artifact/src/test/java/org/mule/runtime/module/artifact/classloader/IBMMQResourceReleaserTestCase.java

```diff
@@ -519,27 +456,16 @@ private int countJULKnownLevels(ClassLoader artifactClassLoader) throws Exceptio
   }
 
   private int countWorkerThreads(ClassLoader artifactClassLoader) {
-    ThreadUtils.getAllThreads().stream().forEach(o -> {
-      System.out.println(o.getName());
-    });
-
-    List<Thread> threads = ThreadUtils.getAllThreads().stream()
+    List<Thread> threads = getAllThreads().stream()
         .filter(thread -> thread.getName().equals(JMSCC_THREAD_POOL_MAIN_NAME))
         .filter(thread -> thread.getClass().getClassLoader() == artifactClassLoader)
         .collect(Collectors.toList());
+
     return threads.size();
   }
 
   private void createWorkerThread(MuleArtifactClassLoader artifactClassLoader) throws ClassNotFoundException {
-    // Load all the clases need to create the IBM Worker thread.
     artifactClassLoader.loadClass(IBM_WORKER_CLASS);
```

Comment:
Not needed, you are already doing the `Class#forName` which does this and also initializes the class proactively.
Try it out. I removed it and the test ran fine.